sysutils/bd/Portfile

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+# Using the latest commit, since there was
+# a fix for Mac OSX after the release.
+github.setup        vigneshwaranr bd c497fe7d
+
+version             1.0.2
+revision            0
+
+categories          sysutils
+platforms           darwin
+license             MIT
+maintainers         nomaintainer
+
+description         a tool to cd into parent directories, instead of ../../.., etc
+long_description    Quickly go back to a specific parent directory in bash \
+                    instead of typing \"cd ../../..\" redundantly.
+
+checksums           rmd160  7c65faa1995763309454b2ab5e879aa150af9697 \
+                    sha256  37e72b32d8dcb65951c7239a1c665fcd9efee144299405621b65c42ecfd2caaa \
+                    size    143089
+
+depends_lib-append  port:bash-completion
+
+use_configure       no
+build {}
+
+destroot {
```

Comment:
With no configure or build phase and an overridden destroot phase, I assume this port installs only architecture-independent files, so the port should declare that with:
```
supported_archs noarch
```

Reply:
That's is correct. There's `bd` + its bash-completion file, and some documentation.

Done √